.gitignore

```diff
@@ -1,4 +1,5 @@
 subbots_python/
+linux_v1.4.0/
```

Comment:
Just out of curiosity, what's this for?

Reply:
So there's a Python application called labelImg that's used for image annotation, you can install it using pip but I've had mixed results installing the pip package because it can mess with your system packages. So I found a place where the binary versions are listed and I just download that one because it seems to be the most stable. Sadly the folder is just called this generic Linux file but the labelImg binary is inside.